src/components/pages/login-page/components/login-form-block/LoginFormBlock.tsx

```diff
@@ -43,34 +44,30 @@ const RightBlock = () => {
           fill
         />
       </Link>
-      <h3 className={styles['register-header']}>З поверненням!</h3>
+      <Typography variant="h3" sx={sxStyles.loginHeader}>
```

Comment:
as your font doesn't changes with width size, remove styles for fonts in `loginHeader` and set correct `variant` for `Typography`